masterlist.yaml

```
@@ -26760,3 +26760,33 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0x76F44BFE
         util: 'SSEEdit v4.0.4c'
+        
+  - name: 'Solitude Docks.esp'
```

There's some extra whitespace here:
```suggestion
        util: 'SSEEdit v4.0.4c'

  - name: 'Solitude Docks.esp'
```